src/persist-client/src/error.rs

```diff
@@ -78,15 +78,15 @@ pub enum InvalidUsage<T> {
         /// The upper bound given to the append call
         append_upper: Antichain<T>,
     },
-    /// An update was not beyond the expected lower of the batch
-    UpdateNotBeyondLower {
+    /// An update was not at or beyond the expected lower of the batch
+    UpdateNotAtOrBeyondLower {
```

Comment:
super nit: I think timely (and Frank) already understand "beyond" as "not less than", which means "at or greater", in laymans terms.